docs/concepts/02-nodes.md

```diff
@@ -72,7 +72,7 @@ const quote = {
 }
 ```
 
-It's important to note that you can use _any_ custom properties you want. The `type` property in that example isn't something Slate knows or cares about. If you were defining your own "link" nodes, you might have a `url` property:
+It's important to note that you can use _any_ custom properties you want. The `type` property in that example isn't something Slate knows or cares about. If you were defining your own "link" nodes, you might have a `URL property:
```

Comment:
This one is missing the trailing quote... regardless I would leave this as `url` as that matches the code example.